### DISCUSSION TITLE
Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities

### DISCUSSION THREAD

**Discussion by MincePie** [vercel/next.js#53972] [Help]
### Summary

I am trying to figure out why Vercel fails my deployments on build.

I pass all linting on VS Code before I push to github (which is used for deployment on Vercel).

My error  message is: Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities

By way of example, line 39 of the file this complains about has:
```

<p className="mt-4 text-md text-gray-700">
              Cookies are small pieces of text sent to your web browser by a website you visit. A cookie file is stored in your web browser and allows the Service or a third-party to recognize you and make your next visit easier and the Service more useful to you. Cookies can be persistent or session cookies. Persistent cookies remain on your personal computer or mobile device when you go offline, while session cookies are deleted as soon as you close your web browser.

            </p>
```

I think I have not used single quotes.

I have also tried to get around this error with amendments to my eslintrc.json 

```
{
  "extends": "next/core-web-vitals",
  "rules": {
    "react/no-unescaped-entities": "off",
    "@typescript-eslint/quotes": "off",
    "quotes": [0],
    "avoidEscape": 0,
    "allowTemplateLiterals": 0,
    "no-useless-escape": 0
    
  }
}
```

I can't find a way to satisfy Vercel that this code can be deployed. It deploys fine on netlfiy.

Can anyone help with next steps?

thank you

**Comment by icyJoseph** [comment]
What do you get if, locally, you run `npx next lint`?

**Comment by victornwakpa** [comment]
> ### Summary
> I am trying to figure out why Vercel fails my deployments on build.
> 
> I pass all linting on VS Code before I push to github (which is used for deployment on Vercel).
> 
> My error message is: Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`. react/no-unescaped-entities
> 
> By way of example, line 39 of the file this complains about has:
> 
> ```
> 
> <p className="mt-4 text-md text-gray-700">
>               Cookies are small pieces of text sent to your web browser by a website you visit. A cookie file is stored in your web browser and allows the Service or a third-party to recognize you and make your next visit easier and the Service more useful to you. Cookies can be persistent or session cookies. Persistent cookies remain on your personal computer or mobile device when you go offline, while session cookies are deleted as soon as you close your web browser.
> 
>             </p>
> ```
> 
> I think I have not used single quotes.
> 
> I have also tried to get around this error with amendments to my eslintrc.json
> 
> ```
> {
>   "extends": "next/core-web-vitals",
>   "rules": {
>     "react/no-unescaped-entities": "off",
>     "@typescript-eslint/quotes": "off",
>     "quotes": [0],
>     "avoidEscape": 0,
>     "allowTemplateLiterals": 0,
>     "no-useless-escape": 0
>     
>   }
> }
> ```
> 
> I can't find a way to satisfy Vercel that this code can be deployed. It deploys fine on netlfiy.
> 
> Can anyone help with next steps?
> 
> thank you
> 
> ### Additional information
> _No response_
> 
> ### Example
> _No response_

I added this in my eslintrc.json and it worked:
`"rules": {
    "react/no-unescaped-entities": "off"
  }`

**Reply by cptiwari20** [reply]
Saved a ton of time, Thanks

**Reply by rugo-cafe** [reply]
thanks man

**Reply by collectivewinca** [reply]
thanks bro

**Reply by AMAZINGMAN2** [reply]
thanks 👍

**Reply by Tautorn** [reply]
Thank you!

**Reply by Recentlystarted** [reply]
Thanks bro, saved my lots of time.

**Reply by SangeethRaajA** [reply]
thank you! [https://nextjs.org/docs/app/api-reference/config/eslint#disabling-rules](url)
